src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -154,6 +154,9 @@ private void LoadDynamoView()
             splashScreen.DynamoView = new DynamoView(viewModel);
             splashScreen.OnRequestStaticSplashScreen();
 
+            //When a xml preferences settings file is located at C:\ProgramData\Dynamo will be read and deserialized so the settings can be set correctly.
+            splashScreen.LoadPreferencesFileAtStartup();
```

Comment:
@QilongTang here my comments:
1. No, I was under the idea that this is a external(or different) import process than splashscreen, I mean, If the user puts a xml file in C:\ProgramData\Dynamo, then opens Dynamo the splashscreen will be shown and the settings will be loaded automatically (automatic process), if the user in splashscreen clicks the "Import Settings"button and select a xml file then it will overwrite the settings already set by the automatic process (let me know if this is the expected behavior).

2. Yes, I will move this line to the SplashScreen.xaml.cs event handler.

Reply:
@QilongTang I moved the method call to splashscreen.
commit: https://github.com/DynamoDS/Dynamo/pull/13500/commits/3c6c3805bddf79b9bb586356864c4502983e1233